package.json

```diff
@@ -17,7 +17,7 @@
     "build:cjs": "node_modules/.bin/rollup -c --environment NODE_ENV:production,TARGET:cjs",
     "build:umd": "node_modules/.bin/rollup -c --environment NODE_ENV:production,TARGET:umd",
     "build:esm": "node_modules/.bin/rollup -c --environment NODE_ENV:production,TARGET:esm",
-    "build:types": "ttsc",
+    "build:types": "ttsc --skipLibCheck",
```

Comment:
Skips library check according to [this issue](https://github.com/Microsoft/TypeScript/issues/14293#issuecomment-282320279). Otherwise types compilation produces a lot of garbage from `node_modules` types.